gradio/_simple_templates/simpletextbox.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import annotations
+
+from typing import Any, Callable
+
+from gradio.components.base import FormComponent
+from gradio.events import Events
+
+
+class SimpleTextbox(FormComponent):
+    """
+    Creates a textarea for user to enter string input or display string output.
```

Comment:
Since we don't support multiple lines anymore, it would be good to replace "textrea" with "textbox" throughout the docstring